src/components/Icons/Svg/Massa/MassaLogo.tsx

```diff
@@ -1,6 +1,18 @@
-export function MassaLogo({ ...props }) {
+type SVGProps = {
+  size?: number;
+  className?: string;
+  rest?: any;
```

Comment:
You don't need to send the `rest` and `className`.
Normally, this is done automatically ?

Reply:
Done!